src/swsssdk/configdb.py

```diff
@@ -2,29 +2,83 @@
 SONiC ConfigDB connection module 
 
 Example:
+    # Write to config DB
     config_db = ConfigDBConnector()
     config_db.connect()
-    config_db.set_table('bgp_neighbor', '10.0.0.1', {
+    config_db.set_entry('BGP_NEIGHBOR', '10.0.0.1', {
         'admin_status': state
         })
 
-TODO:
-    Will add API on keyspace subscription for incremental configuration support
-    in the incoming versions.
+    # Daemon to watch config change in certain table:
+    config_db = ConfigDBConnector()
+    handler = lambda table, key, data: print (key, data)
+    config_db.add_handler('BGP_NEIGHBOR', handler)
+    config_db.connect()
+    config_db.listen()
+
 """
 
 from .dbconnector import SonicV2Connector
 
 class ConfigDBConnector(SonicV2Connector):
 
     def __init__(self):
-        """Connect to Redis through TCP, which does not requires root.
-        """
+        # Connect to Redis through TCP, which does not requires root.
         super(ConfigDBConnector, self).__init__(host='127.0.0.1')
+        self.handlers = {}
 
     def connect(self):
         SonicV2Connector.connect(self, self.CONFIG_DB, False)
 
+    def add_handler(self, table, handler):
```

Comment:
maybe rename to subscribe(self, table, handler) is more common.

I am not sure if we need to add multiple handers to a same table. Do we have such use caes? if we do not need to, then the api signature becomes simpler

subscribe(table, handler)
unsubscribe(table)